LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Colby Miller
```

Comment:
It would be wise to change this to "Copyright (c) 2020 IOMechs" if this code doesn't actually belong to Colby Miller.

Reply:
Oh sorry my bad!